driver/modern_bpf/maps/maps.h

```diff
@@ -56,12 +56,35 @@ __weak const volatile uint64_t probe_schema_var = PPM_SCHEMA_CURRENT_VERSION;
  */
 __weak bool g_64bit_interesting_syscalls_table[SYSCALL_TABLE_SIZE];
 
+/**
+ * @brief Given the syscall id on 64-bit-architectures returns:
+ * - `UF_NEVER_DROP` if the syscall must not be dropped in the sampling logic.
+ * - `UF_ALWAYS_DROP` if the syscall must always be dropped in the sampling logic.
+ * - `UF_NONE` if we drop the syscall depends on the sampling ratio. 
+ */
+__weak uint8_t g_64bit_sampling_syscall_table[SYSCALL_TABLE_SIZE];
+
+/**
+ * @brief Given the tracepoint enum returns:
+ * - `UF_NEVER_DROP` if the syscall must not be dropped in the sampling logic.
+ * - `UF_ALWAYS_DROP` if the syscall must always be dropped in the sampling logic.
+ * - `UF_NONE` if we drop the syscall depends on the sampling ratio. 
+ */
+/// TOOD: we need to change the dimension! we need to create a dedicated enum for tracepoints!
+__weak uint8_t g_64bit_sampling_tracepoint_table[PPM_EVENT_MAX];
```

Comment:
`TP_VAL_MAX`?

Reply:
uhm, let's say probably we will change this tracepoint enum before the next release and right now these `TP_VAL_MAX` are not included in the modern bpf probe, so probably we can change this logic just one time when we are ready, WDYT?

Reply:
Makes sense!